### DISCUSSION TITLE
How to handle commas used instead of decimal points?

### DISCUSSION THREAD

**Discussion by Guiriguanche** [saulpw/visidata#858] [Q&A]
In my locale the decimal "point" is the comma. Does Visidata have a way of handling this? Columns with currencies or floats are not sorted correctly with a "," instead of a ".".

I tried finding an option in the local or global options sheet but was not successful.

**Comment by anjakefala** [comment]
Hi @Guiriguanche, would you mind opening an issue for further investigation?

We have a locale option, that should handle this correctly, but it does not seem to.

**Reply by anjakefala** [reply]
Out of curiousity, what is your LOCALE set to?

What do other programs that you use do to solve this problem?

**Reply by Guiriguanche** [reply]
I opened an issue. It probably is a LOCALE problem. I posted more info in the issue.

**Reply by jabellcu** [reply]
Any news? Would you please point to the issue you opened? Thanks.

**Reply by midichef** [reply]
@jabellcu 
The issue for typing numbers by locale [was solved by the `type-floatlocale` command](https://github.com/saulpw/visidata/issues/863#issuecomment-762006753). (I was able to find the issue by using Github's search feature on the visidata repo, for [issues with `Guiriguanche`](https://github.com/search?q=repo%3Asaulpw%2Fvisidata+guiriguanche&type=issues))

**Reply by jabellcu** [reply]
Ah! True, [issue #863](https://github.com/saulpw/visidata/issues/863#issue-778084965) 
Thank you!
